pkgs/by-name/ra/rav1e/package.nix

```diff
@@ -23,30 +20,26 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-Db7qb7HBAy6lniIiN07iEzURmbfNtuhmgJRv7OUagUM=";
   };
 
-  # update built to be able to use the system libgit2
-  cargoPatches = [ ./update-built.diff ];
-  cargoHash = "sha256-Ud9Vw31y8nLo0aC3j7XY1+mN/pRvH9gJ0uIq73hKy3Y=";
-
-  depsBuildBuild = [
-    pkg-config
-    libgit2
-    zlib
-  ];
+  cargoHash = "sha256-VyQ6n2kIJ7OjK6Xlf0T0GNsBvgESRETzKZDZzAn8ZuY=";
 
   nativeBuildInputs = [
     cargo-c
     nasm
   ];
 
-  env.LIBGIT2_NO_VENDOR = 1;
+  postPatch =
+    ''
+      # remove feature that requires libgit2 and is only used to print a version string
+      substituteInPlace Cargo.toml --replace-fail '"git_version",' ""
```

Comment:
This is from the previous patch so I'm not going to block on it, but I think it would be nice to properly disable this through Cargo

```nix
buildNotDefaultFeatures = true;
buildFeatures = [ "asm" "binaries" "signal_support" "threading" ];
```

Reply:
so the `substituteInPlace` was chosen over `buildFeatures` as that will not cause issues if the underlying `buildFeatures` change.

Reply:
Yeah that's fair

I think it would be better to do something less destructive in the future (in case people do want to use the versions of libraries upstream vendors), but we also don't really have the regular override options for features currently, so that can all be revisited later